assets/queries/cloudFormation/s3_bucket_should_have_bucket_policy/query.rego

```diff
@@ -5,13 +5,14 @@ CxPolicy[result] {
 	resource.Type == "AWS::S3::Bucket"
 	bucket := resource.Properties.BucketName
 	not bucketName(bucket)
-
+    not bucketName(name)
```

Comment:
This should be duplicated?

Reply:
Yes. One checks the name in the property "BucketName", the other checks the name of resource itself.